src/bot/server/handlers/responsive_ui_handlers.py

```diff
@@ -0,0 +1,242 @@
+import json
+
+import bot.server.handlers.constants as constants
+from bot.server.router import callback_handler
+from services.telegram.message_entities_builder import MessageEntitiesBuilder
+from services.telegram.message_manager import MAX_MESSAGE_LENGTH
+
+
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_ADD_ME)
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_REMOVE_ME)
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_CROSS_OUT)
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_UNCROSS_OUT)
+@callback_handler(constants.ButtonCallbackType.RESPONSIVE_UI_REFRESH)
+def handle_generic_responsive_ui_callback(handler_context, update_context):
```

Comment:
We need to resolve performance issues before merging this in

Reply:
Telegram resolved our problem, ty.